Sources/SwiftKuery/Column.swift

```diff
@@ -59,18 +77,30 @@ public class Column: Field, IndexColumn {
         return _table
     }
     
-    /// Initialize an instance of Column.
-    ///
+    // MARK: Column Initializer
```

Comment:
LINE 69 - /// The expression to check for values inserted into of the column.
I don't know what this checkExpression parameter is for but this is not English.